probe/controls/pipes.go

```diff
@@ -0,0 +1,34 @@
+package controls
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+
+	"github.com/weaveworks/scope/xfer"
+)
+
+const (
+	maxOutstandingPIOs = 32
+)
+
+var (
+	// Client is the thing the probe uses to make pipe connections.
+	Client PipeClient
+)
+
+// PipeClient is the thing the probe uses to make pipe connections.
```

Comment:
Is `PipeClient` "the thing the probe uses to make pipe connections", or is that `Client`?